addons/mail/static/tests/discuss/core/composer.test.js

```diff
@@ -49,6 +49,18 @@ test('do not send typing notification on typing after selecting suggestion from
     await assertSteps([]); // No rpc done"
 });
 
+test("typing notification is send when selecting an emoji", async () => {
```

Comment:
Would just name it `send is_typing on adding emoji`
(this is briefer, and at a glance we get faster what's the intent of test)